README.md

```diff
@@ -14,6 +18,38 @@
+  3. IN REVIEW: If a PR is open and a review has been requested (see the [GitHub Workflow](#github) below), the corresponding ticket should be in this column. Similarly, this column may be used for tickets in a QA process (if a separate QA column is not present). Tickets under refactoring or revision based on feedback should move back to IN PROGRESS.
+  4. DONE: Tickets should move to a column indicating completion OR be removed from the view/board after the Review/QA process is completed in order to reduce clutter and facilitate the tracking of total completion time.
+* You are responsible for **managing your own tickets**. This is the primary means of communicating progress with your team and the client—if you don't have an IN PROGRESS ticket we will safely assume you've just moved something into IN REVIEW and assigned it appropriately or are reviewing other tickets assigned to you.
+* Tickets in JIRA should include references/links to their corresponding GitHub PRs or branches, and vice-versa. It is important for review purposes to navigate between the ticket and its implementation quickly and efficiently.
+* If a ticket seems larger than is appropriate, engage the Reporter for clarification or with a recommendation to decompose the ticket into smaller tasks.
+* Engage the Reporter as early as possible if the scope changes on a ticket so a new ticket can be created or relevant parties can be informed about the corresponding timeline/expectation change.
+* Do not assume there is nothing to do. Sometimes project leads or clients fall behind in creating or prioritizing tickets. The first place to look for tickets is the TODO column, the second is the top of the backlog (to work ahead on estimation/clarification), and if neither has tickets that need review please communicate with the team lead.
+* Establish velocity in order to determine a reasonable and sustainable pace. The purpose of estimation is to establish expectations both for delivery timeline and developer productivity. The latter is important if we are to align incentives—ideally extra hours worked or efficiency gains are recognized and rewarded instead of simply leading to higher expectations and a less sustainable workload. No one likes a moving goalpost.
+
+### GitHub Workflow
+
+*  Assign Pull Requests to whomever is immediately responsible for the completion of the ticket. If you are still working on the code, you should be the assignee. If you need a review from someone to move forward, **reassign the PR to that reviewer**.
+* Assigned reviewers (usually a team lead or an in-house client developer) should review the PR in a timely fashion or explicitly indicate they will be delayed and potentially re-assign elsewhere or decline to review. The reviewer must **merge or reassign appropriately** after reviewing.
```

Comment:
@hiattp should we mention that only one reviewer assigned at a time?

Reply:
I don't think we have that constraint.

Reply:
oh, I just wanted to bring this because @hiattp did mention this to me quite a lot, so just in case we needed this :)

Reply:
@sasitornamy yes thanks, we need to make sure that there is only one **mandatory** reviewer assigned. Multiple optional reviewers are probably ok as long as people are being conscientious and sensible about it (not just assigning a bunch of people every time)